denis/configure.py

```diff
@@ -49,7 +49,10 @@ def add_final(parser, required=True):
     remove_parser = command_parsers.add_parser('remove')
     add_assignment(remove_parser)
 
-    command_parsers.add_parser('dump')
+    command_parsers.add_parser('dump').add_argument('-i', '--iso',
+                                                    action='store_true',
+                                                    dest='fmt_iso',
+                                                    help='Dump dates in ISO format')
```

Comment:
I think it would be better to split this on to a second line (i.e. `dump_parser = command_parsers.add_parser...` and then `dump_parser.add_argument...` to be consistent with the other ones